src/scripts/live.js

```diff
@@ -1,119 +1,177 @@
-var events
-var fridayEvents
-var saturdayEvents
-var sundayEvents
+var leaves;
 
-var template
-var lastHalf
-var titleDate = document.getElementById('title-date')
-var timetable = document.getElementById('timetable')
+// Testing flag, for mocking the dates of the events
+// without having to change the dates in the events.json
+var testing = true;
+var first = true;
```

Comment:
the testing variable skips the dates of the events.json and creates the events in the near future for testing purposes